### DISCUSSION TITLE
agency-swarm genesis

### DISCUSSION THREAD

**Discussion by gpu7** [VRSEN/agency-swarm#139] [Q&A] [answered]
Hi, while running "agency-swarm genesis" it aborted with this message:

Exception: ('OpenAI Run Failed. Error: ', 'Rate limit reached for gpt-4o in organization org-xxx on tokens per min (TPM): Limit 30000, Used 19637, Requested 10868. Please try again in 1.009s. Visit https://platform.openai.com/account/rate-limits to learn more.')

It says:  Please try again ...

How do we continue?  Should we run the agency-swarm command again?

**Comment by dwitten392** [comment, marked as answer]
This isn't an ideal answer, but I was able to buy more OpenAI API credits ($50 total), and my rate limit was increased enough to not have rate limit errors.

**Reply by EvanSafiri** [reply]
how much do they increase your rate ?
